.github/workflows/ci.yml

```diff
@@ -33,6 +33,8 @@ jobs:
         ./scripts/clang-tidy-diff.sh
     - name: gen-s-parser
       run: ./scripts/gen-s-parser.py | diff src/gen-s-parser.inc -
+    - name: gen-exprs
```

Comment:
If our CMake code is correct, it shouldn't be necessary to run these steps before CMake time, should it?

Reply:
Good point, I guess running the script is redundant here, I didn't realize that.

But thinking on it, I do still want to run it, just to avoid the risk of a future change in the other tests from affecting this. By running the script explicitly, we get a full self-contained test on one line here.